inc/templates/edit-form.tpl.php

```diff
@@ -134,3 +134,16 @@
 		<# } #>
 	</div>
 </script>
+
+<script type="text/html" id="tmpl-shortcode-ui-content">
+	<div class="field-block">
+		<div class="content-display-container">
+			<p class="content-display">{{ data.shortcode_content }}</p>
+			<a href="#" class="button-edit">Edit Content</a>
+		</div>
+		<div class="content-edit-container" style="display: none;">
+			<textarea class="content-edit">{{ data.shortcode_content }}</textarea>
+			<a href="#" class="button-save">Save Content</a>
```

Comment:
We don't use the edit/save UX for other fields. I'm not sure that inner content should be different. I think it would be nicer if it just live updated like the other fields.

We can probably re-use the same views we use for other fields. It might be nice if it was customizable too - as a textarea might not always be appropriate.